papas/datatypes/Cluster.h

```diff
@@ -26,7 +26,7 @@ class Cluster {
    @param[in]  id identifier type of cluster eg kEcalCluster or kHcalCluster
    @param[in]  subtype single char describing type of cluster eg s = smeared, t= true, m = merged
     */
-  Cluster(double energy, const TVector3& position, double size_m, unsigned int index, IdCoder::ItemType id,
+  Cluster(double energy, const TVector3& position, double size_m, uint32_t index, IdCoder::ItemType id,
```

Comment:
The index is used as a component of the Id using bitshifts etc. I wondered if the bug on lxplus, could have anything to do with the Id (eg writing off the end somehow) so tried to be extra careful. I believe that unsigned int allows for different size implementation on different systems. The Id is fully specified as uint64_t and so I decided it would be safest to also be fully explicit as to the size of the index part of this so that it is guaranteed to be the same on all operating systems.